pages/linux/ipcalc.md

```diff
@@ -0,0 +1,23 @@
+# ipcalc
+
+> Perform simple operations and calculations on IP addresses and networks.
+
+- Show information about an address or network:
+
+`ipcalc {{1.2.3.4}} {{255.255.255.0}}`
```

Comment:
I am a bit unclear on this. The description says - "address or network". But the example contains a subnet mask. Should we also say - "address or network with a given subnet mask" ?

Reply:
@agnivade I've added the suggested language. It definitely helped with clarification.